stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/coil/StreamImageLoaderFactory.kt

```diff
@@ -48,17 +53,23 @@ public class StreamImageLoaderFactory(
                 val dispatcher = Dispatcher().apply { maxRequestsPerHost = maxRequests }
```

Comment:
we don't need `val cacheControlInterceptor` anymore. Or do we?

Reply:
The disk and memory cache are now done through two separate functions, so no longer is an interceptor required. (from what I understand)